play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java

```diff
@@ -36,19 +34,11 @@ public class EbeanDynamicEvolutions extends DynamicEvolutions {
 
   @Inject
   public EbeanDynamicEvolutions(
-      EbeanConfig config,
-      Environment environment,
-      ApplicationLifecycle lifecycle,
-      EvolutionsConfig evolutionsConfig) {
```

Comment:
Can you please keep this old constructor and deprecate it? Just call the new constructor from it, the `ApplicationLifecycle` will be unused but that's ok.

Reply:
O.K, I put the @Inject annotation on the new constructor and @Deprecated on the previous one.